app/controllers/application_controller.rb

```diff
@@ -17,6 +17,10 @@ def search
     @search = Vehicle.ransack
   end
 
+  def load_data
+    @branchs = Branch.select :id, :name
```

Comment:
cái thằng branch này có nhiều data k e?

Reply:
Không ạ, có mỗi truoừng name với mấy trường time thôi ạ

Reply:
ý chị là nhiều record k đó

Reply:
Ít thôi ạ

Reply:
Nếu nhiều record thì mình không thể để này. Nó sẽ lấy toàn bộ record ra ảnh hướng lớn tới performent đó.
Nhiều recort thì sẽ chỉ hiện thị giới hạn hoặc sẽ cho người dùng xem ở 1 trạng khác có phân trang và search chẳng hạn.